draft-ietf-quic-transport.md

```diff
@@ -4590,8 +4595,14 @@ preferred_address (0x000d):
 active_connection_id_limit (0x000e):
 
 : The maximum number of connection IDs from the peer that an endpoint is willing
-  to store. This value includes only connection IDs sent in NEW_CONNECTION_ID
-  frames. If this parameter is absent, a default of 0 is assumed.
+  to store. This value includes connection IDs sent in NEW_CONNECTION_ID frames,
+  the connection ID used during the handshake as well as the connection ID
+  provided in the preferred_address transport parameter.
+  Unless a zero-length connection ID is being used, the value of the
+  active_connection_id_limit parameter MUST be no less than 2. If this
+  transport parameter is absent, a default of 2 is assumed.
+  When a zero-length connection ID is being used, the active_connection_id_limit
+  parameter MUST NOT be sent.
```

Comment:
I think this change is unnecessary and detrimental. Why is this useful?

Reply:
Can you elaborate which part you find detrimental?

Reply:
@janaiyengar PTAL at https://github.com/quicwg/base-drafts/pull/3201#discussion_r347122022.

Reply:
(Would you rather it was ignored rather than not sent?)

Reply:
I said detrimental because we used to not encode values on the wire that were illegal ... but we've walked away from that in other places too :-/ I'm ok with this.